plugin/pkg/admission/security/podsecurity/testdata/pod_baseline.yaml

```diff
@@ -0,0 +1,670 @@
+# this pod fixture is used for benchmarks and should be kept updated to pass the latest baseline policy
```

Comment:
Rename `pod.yaml` to `pod_restricted.yaml`?

Reply:
done